shared/middleware/properties_api.rb

```diff
@@ -74,7 +74,12 @@ class PropertiesApi < Sinatra::Base
 
     _, decrypted_channel_id = storage_decrypt_channel_id(channel_id)
     parsed_value = PropertyBag.parse_value(request.body.read)
-    value = PropertyType.new(decrypted_channel_id, storage_id(endpoint)).set(name, parsed_value, request.ip)
+    begin
```

Comment:
Yeah, wouldn't actually need any changes to properties_api.rb with the current design (I was making sure it actually got the message from my class).